### ISSUE TITLE
right click not working

### ISSUE THREAD

**Issue by Preston2jager** [Eugeny/tabby#9414]
Ubuntu 22.04
Tabby 1.0.205

Fresh installation 

Right-click does not work. I have tried to set it as "paste“ or "context menu" but nothing happens.
middle click paste is working fine.
ctrl-shift-v paste is working fine

**Comment by renpj** [comment]
I have the same problem on LinuxMint after upgrade to LinuxMint 21.3.

**Comment by jlecointecaplaser** [comment]
Same problem here on LinuxMint 21.3

**Comment by sheinrich177** [comment]
I had this issue suddenly occurring in tabby version 1.0.207 under Linux Mint 21.3. I was able to fix it by uninstalling tabby, rebooting, removing `~/.config/tabby` folder and reinstalling tabby.

 I lost my tabby settings, but right-click context menu is back.

**Comment by jlecointecaplaser** [comment]
Great, it worked, thanks !

It's possible to save and re-import the config file to avoid losing anything  `~/.config/tabby/config.yaml`

**Comment by sheinrich177** [comment]
Good point, I missed that. It should be possible to backup and restore the config.yaml file, but I haven't tried it yet.

Problems could actually only occur if a newer version of Tabby is installed than previously installed, e.g. if Tabby Team changes some configuration keys, ...)

**Comment by jlecointecaplaser** [comment]
I must admit I sweated a little when re-importing my conf file

**Comment by tinywolf3** [comment]
> I had this issue suddenly occurring in tabby version 1.0.207 under Linux Mint 21.3. I was able to fix it by uninstalling tabby, rebooting, removing `~/.config/tabby` folder and reinstalling tabby.
> 
> I lost my tabby settings, but right-click context menu is back.

it worked. thank you very much.

**Comment by satacata** [comment]
This is so annoying..is this bug still open? Everything was working fine, but suddenly noticed that right click no longer works..Restoring the config file does not help. I am shocked to see there is no perfect terminal emulator!!

**Comment by kikyoulg** [comment]
mac has same issue ，but can't find this dir

**Comment by Chealer** [comment]
@Preston2jager: Does this persist in Tabby 1.0.230 or above? And if so, which desktop environment do you use?
